tests/case-01/runtest

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+
+# Python script to run and analyse a self-convergence test
+
+from __future__ import division
+from __future__ import print_function
+
+try:
+  from builtins import str
+except:
+  pass
+
+from boututils.run_wrapper import shell, launch, getmpirun
+from boutdata.collect import collect
+
+from numpy import sqrt, max, abs, mean, array, log, concatenate
+
+# List of NY values to use
+nylist = [20, 40, 80, 160, 320]
```

Comment:
only the last 3 ones are actually used in the test, not sure what the time overhead is though ...